cmd/run.go

```diff
@@ -34,16 +36,42 @@ var image string
 var enableProfiling bool
 var logLevel string
 var protocol string
+var isDaemon bool
 
 var RunCmd = &cobra.Command{
 	Use:   "run",
 	Short: "Launches Dapr and your app side by side",
```

Comment:
`Launches Dapr and (optionally) your app`?

Reply:
Done.